Sources/Lumina/Camera/Extensions/CapturePhotoExtension.swift

```diff
@@ -9,14 +9,18 @@
 import UIKit
 import AVFoundation
 
-@available (iOS 11.0, *)
 extension AVCapturePhoto {
   func normalizedImage(forCameraPosition position: CameraPosition) -> UIImage? {
     LuminaLogger.notice(message: "normalizing image from AVCapturePhoto instance")
     guard let cgImage = self.cgImageRepresentation() else {
       return nil
     }
-    return UIImage(cgImage: cgImage as! CGImage, scale: 1.0, orientation: getImageOrientation(forCamera: position))
+    #if swift(>=5.5)
+    return UIImage(cgImage: cgImage, scale: 1.0, orientation: getImageOrientation(forCamera: position))
+    #else
+    return UIImage(cgImage: cgImage.takeUnretainedValue(), scale: 1.0, orientation: getImageOrientation(forCamera: position))
+    #endif
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/8cefb8053a7a558007ec4a93a1a2b01854a11a38/Sources/Lumina/Camera/Extensions/CapturePhotoExtension.swift#L23"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>CapturePhotoExtension.swift:23</code></td>
    </tr>
  </tbody>
</table>